lib/styles/default.coffee

```diff
@@ -4,7 +4,7 @@ path = require 'path'
 _            = require 'underscore'
 coffeeScript = require 'coffee-script'
 fsTools      = require 'fs-tools'
-jade         = require 'jade'
+pug         = require 'pug'
```

Comment:
You forgot to adjust the whitespace to preserve the alignment :)

Reply:
Updated